src/libLookup/Lookup.cpp

```diff
@@ -1855,6 +1855,18 @@ void Lookup::RetrieveDSBlocks(vector<DSBlock>& dsBlocks, uint64_t& lowBlockNum,
     highBlockNum = curBlockNum;
   }
 
+  uint64_t diff = 0;
+  if (!partialRetrieve &&
+      SafeMath<uint64_t>::sub(highBlockNum, lowBlockNum, diff)) {
+    if (diff > FETCH_DS_BLOCK_LIMIT) {
```

Comment:
Do we think that we should increase the `FETCH_DS_BLOCK_LIMIT`?  As on the client-side, the getter function `FetchLatestDSBlocksSeed` is called in a loop, So there would be many calls for a large block range and every time lookup has to execute function for sliced block range.